app/(afterLogin)/(deal)/deals/manage-deals/_components/sideBar/index.tsx

```
@@ -0,0 +1,200 @@
+          ></Button>
+        </div>
+
+        <Form.Item>
+        <div className="grid grid-cols-2 gap-4">
+
+          <Select placeholder="Deal Type" />
+          <Input className="border-[#e7e7e8]" placeholder="Role Name" />
+          </div>
+        </Form.Item>
+        <div className="space-y-6">
+      {/* Upload File */}
```

ESLint

[error] Replace `······` with `··········`

(prettier/prettier)